src/main/java/edu/ie3/datamodel/utils/validation/ThermalUnitValidationUtils.java

```diff
@@ -7,11 +7,14 @@
 
 import edu.ie3.datamodel.exceptions.InvalidEntityException;
 import edu.ie3.datamodel.exceptions.ValidationException;
+import edu.ie3.datamodel.models.input.container.ThermalGrid;
 import edu.ie3.datamodel.models.input.thermal.*;
 import edu.ie3.datamodel.utils.Try;
 import edu.ie3.datamodel.utils.Try.Failure;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Set;
+import java.util.stream.Stream;
 import javax.measure.Quantity;
 
 public class ThermalUnitValidationUtils extends ValidationUtils {
```

Comment:
I think it would be better to rename this class, because we can validate more than just `ThermalUnits`. I think a better name would be `ThermalValidationUtils`.